draft-ietf-mops-streaming-opcons.md

```diff
@@ -413,15 +413,41 @@ Here are a few common resolutions used for video content, with typical ranges of
+
+Between these two (potentially competing) "helpful" mechanisms each responding to the same bottleneck with no coordination betten themselves, so that each is unaware of actions taken by the other, it is easily possible for the media server to estimate available path bandwidth to a media player, reduce the amount of bandwidth in use based on that estimation, and then continue to use this lowwe estimated available bandwidth value for some time, resulting in a quality of experience for media consumers that is significantly lower than what could have been achieved. In order to avoid this situation, which can potentially affect all the users whose streaming media traverses a bottleneck link, there are several possible mitigations that streaming operators can use.
+
+### Know Your Network Traffic {#sec-know-your-traffic}
 
-## Path Requirements
+Recognizing that a path carrying streaming media is "not behaving the way it normally does" is fundamental. Analytics that aid in that recognition can be more or less sophisticated, and can be as simple as noticing that the apparent round trip times for media traffic carried over TCP transport on some paths are suddenly and significantly longer than usual. This is possible because monitors can detect how long specific TCP segments are taking to be acknowledged by a TCP receiver, since TCP octet sequence numbers and acknowledgements for those sequence numbers are "carried in the clear", even if the TCP payload itself is encrypted.
+
+There are many reasons why path characteristics might change suddenly, for example,
+
+* "cross traffic" that traverses part of the path, especially if this traffic is "inelastic", and does not, itself, respond to indications of path congestion.
+
+* routing changes, which can happen in normal operation, especially if the new path now includes path segments that are more heavily loaded, offer lower total bandwidth, or simply cover more distance, but can affect media consumer quality of experience even if this happens in "normal operation".
+
+**Open Issue 1** - do we name products or protocols in common use here?
+
+**Open Issue 2** - we probably COULD mention the In-Situ OAM work in IPPM, recognizing that over-the-top streaming media operators are unlikely to be able to use this end-to-end, but at least some streaming media operators may be able to use it over parts of network paths that they can control.
+
+**Open Issue 2** - we probably COULD mention the Qlog work that has now been adopted in the QUIC working group, and is at least somewhat mature (it's certainly in use at scale in places like Facebook), while noting that this is part of the transport protocol evolution topic in {{quic-behavior}}.
```

Comment:
Are there any papers we can reference here on what stats are important here?  I haven't really done much in this space, but my instinct is that if there's a good reference we can point to, outlining the useful stats to collect and good known ways to collect them seems worthwhile, but we should avoid making up as much as we possibly can.
(And regarding the first open issue about whether to mention protocols, I guess if we do wind up pointing to stats approaches, saying what contexts they can work in seems ok, otherwise less is more here I expect.  I don't think people need a list of transport protocols in this section if there's not something in particular we need to attach to them.)

Reply:
I recently read this and liked it: https://www.compiralabs.com/ultimate-guide-congestion-control

Reply:
@acbegen - I do like that paper. Are you thinking we should put it in the "further reading" section, or try to use it to make sure we haven't missed anything in our text, or something else?

Reply:
@GrumpyOldTroll, this is the current state of play in QLog: https://datatracker.ietf.org/doc/search/?name=quic-qlog&activedrafts=on.

If I write something that accurately describes that state of play, do you think that's OK? And if it's OK, is it sufficient?

Reply:
And yes, I will add this to the reading list.